<?xml version="1.0"?>
<launch>

<!-- Scene Graph Generator  -->
  <node pkg="turtlebot2i_safety" name="sg_generator" type="sg_generator_for_labeling.py" output="log"/>

<!-- Scene Graph Parser  parse_sg_for_labeling.py --> 
  <node pkg="turtlebot2i_safety" name="parse_sg" type="parse_sg.py" output="screen"/>

</launch>
